index.html

```diff
@@ -383,6 +383,58 @@ <h1 class="banner__title">
           </div>
         </div>
       </section>
+
+      <!-- Block program -->
+
+      <section class="program">
+        <div class="container">
+          <div class="row">
+            <div class="column-lg-9">
+              <div class="program__text">
+                <div class="program__title">Программа курса</div>
+
+                <div class="program__square-front">
```

Comment:
check these articles to implement accordion
https://doka.guide/html/details/
https://nikitahl.com/native-html-accordion

Reply:
+